worker/codalabworker/docker_utils.py

```diff
@@ -0,0 +1,289 @@
+import logging
+import os
+import docker
+
```

Comment:
Would be nice to add a block comment up here with respect to how this class is to be used, what functionality it provides over the Docker API.

Reply:
Oh true.
TODO: Add module doc for `docker_utils`